src/main/java/com/minecolonies/colony/buildings/BuildingLumberjack.java

```diff
@@ -8,47 +8,94 @@
 import com.minecolonies.colony.jobs.JobLumberjack;
 import net.minecraft.util.BlockPos;
 
+/**
+ * The lumberjacks building.
+ */
 public class BuildingLumberjack extends AbstractBuildingWorker
 {
+    /**
+     * The maximum upgrade of the building.
+     */
+    private final static int MAX_BUILDING_LEVEL = 3;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)